source/models/customizing-serializers.md

```diff
@@ -135,7 +136,40 @@ import DS from 'ember-data';
 export default DS.JSONSerializer.extend({});
 ```
 
-#### ID
+To change the format of the data that is sent to the backend store, you can use
+the `serialize` hook:
+
+```app/serializers/application.js
+import DS from 'ember-data';
+
+export default DS.JSONSerializer.extend({
+  serialize(snapshot, options) {
+    // help! what do i do here?
+  }
+});
+```
+
+If your backend store provides data in a format other than JSON API, you can
+use the `normalizeResponse` hook to change the data into the JSON API format:
+
+```app/serializers/application.js
+import DS from 'ember-data';
+
+export default DS.JSONSerializer.extend({
+  normalizeResponse(store, primaryModelClass, payload, id, requestType) {
```

Comment:
We should really encourage people to use `this._super`.

Basically:
Before you call super, the payload is in the format that the serializer expects. Munge it but keep that format.
After you call super, the payload is now in JSON API format, feel free to munge as long as you keep it JSON API.

Delegating conversion to super gives you A LOT for free + you have the flexibility to make customizations in two different formats.